packer-builds/centos-base/base.json

```diff
@@ -1,6 +1,7 @@
 {
     "variables": {
         "do_api_token": "{{ env `DIGITALOCEAN_API_TOKEN` }}",
+        "do_user": "{{ env `DIGITALOCEAN_USER` }}",
```

Comment:
This can be `SYS_USER`, since it's us making Linux users, not DO users. We can't use just `USER` because that clobbers a common environment variable.